src/views/HomeView.vue

```diff
@@ -20,9 +28,12 @@ import axios from "axios";
 })
 export default class HomeView extends Vue {
   post: IPost | null = null;
-  async mounted() {
-    // await this.getPost();
-  }
+  maxLength = 5000;
+  textareaOptions: object = {};
```

Comment:
Where it is used? Could you avoid `object` type? Use interfaces, simple types, enums... or null if the data/prop may not have the default value

Reply:
not used anywhere yet, left it for the future